test/tz_fake.py

```diff
@@ -0,0 +1,37 @@
+import pytz
+from contextlib import contextmanager
+import time
+import os
+import pytest
+import tzlocal
+
+
+@contextmanager
+def fake_timezone_ctx(tz):
```

Comment:
Maybe instead of `fake_` prefix we could use `stub_` prefix instead to give a better indication what's going on?

Reply:
Done, also changed the filename.